route53.tf

```diff
@@ -1,15 +1,28 @@
+locals {
+  alias = [for a in var.alias : {
+    domain       = a.domain
+    hostname     = a.hostname
```

Comment:
This was **not** intentional; good catch. I have fixed it in the latest push. 

Please note that, if the user specifies more than one alias but omits the hostname argument on both, Route 53 will reject the `apply` because of a duplicate hostname. That's obviously a user error, however.